specification/indices/resolve_cluster/ResolveClusterResponse.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ElasticsearchVersionMinInfo } from '@_types/Base'
+
+export class Response {
+  body: {
```

Comment:
According to the documentation's examples, this should be a a dictionnary with this layout as value.

Something like:
```suggestion
  body: Dictionary<string, ClusterInfo>
```
Even better if we can name the string to give meaning.

Reply:
Good catch. I had the response all wrong. Sorry for that. I've fixed it in the latest push.